site/src/components/DeploySettingsLayout/optionValue.ts

```diff
@@ -19,6 +22,26 @@ export function optionValue(option: ClibaseOption) {
       return Object.entries(option.value as Record<string, string>).map(
         ([key, value]) => `"${key}"->"${value}"`,
       );
+    case "Experiments": {
+      const experimentMap: Record<string, boolean> | undefined =
+        additionalValues?.reduce(
+          (acc, v) => {
+            return { ...acc, [v]: option.value.includes("*") ? true : false };
```

Comment:
Is the max number of items in `additionalValues` expected to be pretty small? The current approach creates a brand new object for each "iteration" of the reduce call, but that won't be a big deal if this isn't going to iterate over much data

Reply:
Definitely small! Otherwise we'd have to figure out a different way to display entirely.